src/types/article.ts

```diff
@@ -160,6 +160,13 @@ export default /* GraphQL */ `
     "Total number of appreciations recieved of this article."
     appreciationsReceivedTotal: Int!
 
+
+    "Total number of donation recieved of this article."
+    donationCount: Int!
+
+    "Total number of readers of this article."
+    readerCount: Int!
```

Comment:
what's the cache TTL? or revalidate on `article_ga4_data` updates?

Reply:
right, here TTL should be set

Reply:
right, here TTL should be set